netlify-plugin-cloudinary/src/index.ts

```diff
@@ -127,6 +127,8 @@ export async function onBuild({
   }
 
   console.log(`[Cloudinary] Using host: ${host}`);
+  console.log('PUBLISH_DIR', constants["PUBLISH_DIR"])
```

Comment:
forgot to ask - were you intentionally leaving the logs here? guessing these were for testing, if so can we clean these out quick? then we can merge 🚀

Reply:
Ack no I did not mean to leave that there 😂 Great catch, thank you!